src/ui/omnibox/Omnibox.qml

```diff
@@ -39,6 +39,11 @@ Item {
     implicitHeight: 40
     implicitWidth: 256
 
+    function focusUrlField() {
```

Comment:
Style: JavaScript function should be declared below property declarations but above property assignments.
See http://doc.qt.io/qt-5/qml-codingconventions.html